site/src/pages/[shortname]/blocks/[block-slug].page.tsx

```diff
@@ -129,9 +131,7 @@ export const getStaticPaths: GetStaticPaths<
   BlockPageQueryParams
 > = async () => {
   return {
-    paths: (await readBlocksFromDisk()).map(
-      (metadata) => metadata.blockPackagePath,
-    ),
+    paths: [],
```

Comment:
Doing this now since 'build every block page' will become infeasible once we have many.